doc/locale/ja/LC_MESSAGES/developer.po

```diff
@@ -157,10 +161,10 @@ msgid "`Groonga <http://groonga.org/>`_"
 msgstr ""
 
 msgid "`Cutter <http://cutter.sourceforge.net/>`_ (for C/C++ unit tests)"
-msgstr "Cutter <http://cutter.sourceforge.net/>`_ (C/C++単体テスト用)"
+msgstr "`Cutter <http://cutter.sourceforge.net/>`_ (C/C++単体テスト用)"
 
 msgid "`Sphinx <http://sphinx-doc.org/>`_ (for documents)"
-msgstr "Sphinx <http://sphinx-doc.org/>`_ (ドキュメント用)"
+msgstr "`Sphinx <http://sphinx-doc.org/>`_ (ドキュメント用)"
```

Comment:
```suggestion
msgstr "`Sphinx <https://sphinx-doc.org/ja/>`_ (ドキュメント用)"
```

Reply:
I have changed these links to `https://www.sphinx-doc.org/en/master/` and  `https://www.sphinx-doc.org/ja/master/` because  `https://sphinx-doc.org/...`  is not found (`http://sphinx-dox.org/...` is redirected to `https://www.sphinx-doc.org/...`, throuh).

BTW, this line will be removed if #352 is merged so I think we don't need to pay much attention about this line.